server/analysis/tests/test_gpr.py

```diff
@@ -67,6 +68,37 @@ def test_gprnp_sigmas(self):
         self.assertEqual(sigmas_round, expected_sigmas)
 
 
+# test GPFlow version GPR
+class TestGPRGPF(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestGPRGPF, cls).setUpClass()
+        boston = datasets.load_boston()
+        data = boston['data']
+        X_train = data[0:500]
+        X_test = data[500:]
+        y_train = boston['target'][0:500].reshape(500, 1)
+
+        model_kwargs = {'lengthscales': 1, 'variance': 1, 'noise_variance': 1}
+        tf.reset_default_graph()
+        graph = tf.get_default_graph()
+        gpflow.reset_default_session(graph=graph)
+        cls.m = gpr_models.create_model('BasicGP', X=X_train, y=y_train,
+                                        **model_kwargs)
+        cls.gpr_result = gpflow_predict(cls.m.model, X_test)
+
+    def test_gprnp_ypreds(self):
```

Comment:
I think all the function names in this file are misleading. We can remove the "_gprnp" part from them.